components/Carousel.js

```diff
@@ -22,3 +22,37 @@
     <div class="right-button"> > </div>
   </div>
 */
+
+
+function carouselBuilder() {
+
+    const carDiv = document.createElement('div')
+    const leftButton = document.createElement('div')
+    const img1 = document.createElement('img')
```

Comment:
you could also create an array of image elements and an array of the src urls and then use .forEach to set the .src property